nextmv/options.py

```diff
@@ -221,9 +360,7 @@ def _parameter_value(parameter: Parameter, value: Any) -> Any:
 
         value = str(value).lower()
 
-        if value in ("true", "1", "t", "y", "yes"):
+        if value in ("true", "True", "TRUE", "1", "t", "T", "y", "Y", "yes", "Yes", "YES"):
             return True
-        if value in ("false", "0", "f", "n", "no"):
-            return False
 
-        raise argparse.ArgumentTypeError(f"invalid value for bool parameter '{parameter.name}': {value}")
+        return False
```

Comment:
Yes, working in a Notebook with `nextmv.Options` and a bool option turned out problematic, so that is why we had to default to `False`, instead of an exception. Sorry.